rsconnect/main.py

```diff
@@ -75,7 +75,7 @@ def test(server, api_key, insecure, cacert, verbose):
 
 
 def _test_server_and_api(server, api_key, insecure, ca_cert):
-    ca_data = ca_cert and ca_cert.read()
+    ca_data = ca_cert and unicode(ca_cert.read())
```

Comment:
This doesn't work under 3.x. Is there a `six` function that can help here?

Reply:
Maybe `six.u(...)`?

Reply:
done